agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/RequestFactory.java

```diff
@@ -104,6 +105,28 @@ private static KeyStore createKeyStore(
     return request;
   }
 
+  /**
+   * Setting applicationID and Signature within the header.
+   *
+   * @param url -
+   * @param applicationId -
+   * @param signature -
+   * @return Builder -
+   */
+  public static Invocation.Builder signedDeleteRequest(
+      String url, String applicationId, String signature) {
+    Client client = ClientBuilder.newClient();
+    client.property(ClientProperties.SUPPRESS_HTTP_COMPLIANCE_VALIDATION, true);
+    client.property(LoggingFeature.LOGGING_FEATURE_LOGGER_LEVEL_CLIENT, "INFO");
```

Comment:
This will cause the client to log every request. This should be something the user should define, not the API.

Reply:
This exact behavior can be found in all the other Requests in this file as well, let's decide on a common behavior for all types of requests and change this for all requests in one common way in an additional pull request.

Reply:
Ok, I'll file an issue for that.